src/name_bazaar/ui/pages/offering_detail_page.cljs

```diff
@@ -157,28 +158,30 @@
     (fn []
       (let [{:keys [:offering/address]} @route-params
             offering-loaded? @(subscribe [:offering/loaded? address])
-            offering @(subscribe [:offering address])]
-        [app-layout
-         [ui/Segment
-          [ui/Grid
-           {:class "layout-grid"}
-           [ui/GridRow
-            [ui/GridColumn
-             {:class :join-lower
-              :computer 8
-              :tablet 8
-              :mobile 16}
-             [:h1.ui.header "Offering " (:offering/name offering)]]
-            [ui/GridColumn
-             {:class :join-lower
-              :computer 8
-              :tablet 8
-              :mobile 16
-              :floated "right"}
-             [share-buttons
-              {:url @(subscribe [:page-share-url :route.offerings/detail {:offering/address address}])
-               :title (str "Check out offering for " (:offering/name offering) " on NameBazaar")}]]]]
-          (if offering-loaded?
-            [offering-detail]
-            [:div.padded [content-placeholder]])]
-         [similar-offerings]]))))
+            {:keys [:offering/name] :as offering} @(subscribe [:offering address])]
+        [:div.wrapper
+         [name-bazaar-meta-tags {:title name :description (str "Details of offering for " name)}]
```

Comment:
:title "[name] Offering/Auction"
:description "[name] is offered on NameBazaar! Price/Minimum bid: [price]"